en/plugins/zapier.md

```diff
@@ -4,12 +4,32 @@ This integration integrates any Mautic instance with 1.100+ third party web serv
 
 ## Requirements
 
-1. Mautic version 2.9.0 or newer with SSL - basic authentication requires HTTPS to be secure.
-3. Zapier account. Free or Paid. [View Zapier pricing](https://zapier.com/pricing/).
+Zapier account. Free or Paid. [View Zapier pricing](https://zapier.com/pricing/).
 
 ## Installation
 
-There is no need to install any additional plugin to your Mautic instance. Zapier works with the API and webhooks already available within Mautic. This Zapier integration use Basic Authentication to access Mautic API. It is disabled by default so enable this first.
+There is no need to install any additional plugin to your Mautic instance. Zapier works with the API and webhooks already available within Mautic.
+
+### Oauth2 Authentication for Zapier 3.x
+
+New version of app improve authentication by Oauth2 protocol.
+
+1. Open the admin menu
+2. Go to Mautic's global configuration > API Settings
+3. Set __API enabled?__ to __Yes__.
+4. Save the configuration.
+5. Go to Mautic's global configuration > API Credentials > __Oauth2__
+6. Create new __Oauth2___ application with redirect URI: __tba__
+7. Copy Public Key and Secret Key of your created app
+
+![How to enable API](https://user-images.githubusercontent.com/462477/74520415-cc616b00-4f17-11ea-8dbe-f6dd8a0a6cfc.png)
+
+![New Oauth2 application](https://user-images.githubusercontent.com/462477/74520342-a1771700-4f17-11ea-866f-5be1c895f0f9.png)
+
+ 
+### Basic Authentication app for Zapier 2.x 
+ 
+ If you're using Zapier 2.x, this Zapier integration use Basic Authentication to access Mautic API. It is disabled by default so enable this first.
```

Comment:
If you are using Zapier 2.x, Basic Authentication is used to access the Mautic API. This is disabled by default, so you will need to enable it first.